### PR TITLE
Add memoryBacking and check_in_vm into libvirt_disk.py

### PR DESCRIPTION
**Pull request by chunfuwen**
Those two APIs can be more of common utils

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.vhostuser.coldplug.raw_format.normal_test.start_vhostuser_vm

**Comment by chunfuwen** [comment]
@kylazhang 
@kamilvarga
This is part of comments addressed from 
https://github.com/autotest/tp-libvirt/pull/3476
and move this APIs into avocado-vt projects

**Review by kamilvarga — APPROVED** [review]
LGTM.

File: virttest/utils_libvirt/libvirt_disk.py [hunk 1]
```diff
@@ -431,3 +432,80 @@ def get_mirror_part_in_xml(vm, disk_target):
...
+                                                      "yum install -y parted", 300)
+        if rpm_stat != 0:
+            raise exceptions.TestFail("Failed to query/install parted:\n%s", out_put)
+
+        new_parts = utils_disk.get_parts_list(session)
+        added_parts = list(set(new_parts).difference(set(old_parts)))
+        logging.info("Added parts:%s", added_parts)
+        if is_equal:
+            if len(added_parts) != 0:
+                return False
```
Comment (kylazhang) [inline virttest/utils_libvirt/libvirt_disk.py:474]: Better give logging.error here as below

**Review by kylazhang — COMMENTED** [review]
Others are ok for me

File: virttest/utils_libvirt/libvirt_disk.py (hunk 1 above)
Comment (kylazhang) [reply virttest/utils_libvirt/libvirt_disk.py:474]: I think should be like "new added parts are not equal the old one"

Comment (chunfuwen) [reply virttest/utils_libvirt/libvirt_disk.py:475]: @kylazhang 
add  logging.error with dumping message likes 'new added parts are not equal the old one'

**Review by kylazhang — APPROVED** [review]

**Pull request merged**